mod_modular_vanilla/hooks/entity/tactical/actor.nut

```diff
@@ -358,10 +358,10 @@
 		{
 			local overflowDamage = _hitInfo.DamageArmor;
 
-			if (this.getBaseProperties().Armor[_hitInfo.BodyPart] != 0)
+			if ((this.getBaseProperties().Armor[_hitInfo.BodyPart] * p.ArmorMult[_hitInfo.BodyPart]) != 0)
 			{
-				overflowDamage -= this.getBaseProperties().Armor[_hitInfo.BodyPart] * this.getBaseProperties().ArmorMult[_hitInfo.BodyPart];
-				this.getBaseProperties().Armor[_hitInfo.BodyPart] = this.Math.max(0, this.getBaseProperties().Armor[_hitInfo.BodyPart] * this.getBaseProperties().ArmorMult[_hitInfo.BodyPart] - _hitInfo.DamageArmor);
+				overflowDamage -= this.getBaseProperties().Armor[_hitInfo.BodyPart] * p.ArmorMult[_hitInfo.BodyPart];
```

Comment:
Vanilla uses `getBaseProperties`, but that is not accurate enough, we need the current properties instead for this calculation.
In Vanilla there is not a single enemy, whose base property ArmorMult is anything other than 1.0.
But Ifrits have a current ArmorMult of 1.0, 2.0 or 4.0 depending on size.

I couldnt explain you how this exactly changes the overflowDamage mechanic. But it is for sure more accurate after this fix for ifrits